Dockerfile.arm

```diff
@@ -35,35 +34,35 @@ ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
 ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
 
 COPY --from=qemu /usr/bin/qemu-arm-static /usr/bin
+COPY --from=builder /jellyfin /jellyfin
+COPY --from=web-builder /dist /jellyfin/jellyfin-web
 RUN apt-get update \
- && apt-get install --no-install-recommends --no-install-suggests -y ca-certificates gnupg curl && \
- curl -ks https://repo.jellyfin.org/debian/jellyfin_team.gpg.key | apt-key add - && \
- curl -s https://keyserver.ubuntu.com/pks/lookup?op=get\&search=0x6587ffd6536b8826e88a62547876ae518cbcf2f2 | apt-key add - && \
- echo 'deb [arch=armhf] https://repo.jellyfin.org/debian buster main' > /etc/apt/sources.list.d/jellyfin.list && \
- echo "deb http://ppa.launchpad.net/ubuntu-raspi2/ppa/ubuntu bionic main">> /etc/apt/sources.list.d/raspbins.list && \
- apt-get update && \
- apt-get install --no-install-recommends --no-install-suggests -y \
- jellyfin-ffmpeg \
- libssl-dev \
- libfontconfig1 \
- libfreetype6 \
- libomxil-bellagio0 \
- libomxil-bellagio-bin \
- libraspberrypi0 \
- vainfo \
- libva2 \
- locales \
- && apt-get remove curl gnupg -y \
+ && apt-get install --no-install-recommends --no-install-suggests -y ca-certificates gnupg wget apt-transport-https \
+ && wget -O - https://repo.jellyfin.org/jellyfin_team.gpg.key | apt-key add - \
+ && echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release ) $( awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release ) main" | tee /etc/apt/sources.list.d/jellyfin.list \
+ && wget -O - https://archive.raspbian.org/raspbian.public.key | apt-key add - \
+ && echo "deb http://archive.raspbian.org/raspbian buster main contrib non-free firmware">> /etc/apt/sources.list.d/raspbins.list \
+ && apt-get update \
+ && apt-get install --no-install-recommends --no-install-suggests -y \
+   jellyfin-ffmpeg \
+   libssl-dev \
+   libfontconfig1 \
+   libfreetype6 \
+   libomxil-bellagio0 \
+   libomxil-bellagio-bin \
+   libraspberrypi0 \
+   vainfo \
+   libva2 \
+   openssl \
```

Comment:
did this build? iirc i had to use `libssl-dev`

Reply:
Yes I finally have automatic builds working: https://hub.docker.com/repository/registry-1.docker.io/rigtorp/jellyfin/tags?page=1